src/components/editor/TextEditor/SlideToolBox.tsx

```diff
@@ -27,7 +27,7 @@ export default function SlideToolBox({
             [styles.item__button]: true,
             [styles['item__button--tool']]: true,
           })}
-          onClick={isShow.toggle}
+          onClick={visible ? hide : show}
```

Comment:
`useBooleanState` 에 toggle을 만들어두신 만큼 활용하는 편이 좋아보여요.

```suggestion
          onClick={toggle}
  // ...선언부에 추가
  // const [visible, show, hide, toggle] = useBooleanState(false);
```